exalearn-ui/src/app/header/header.component.less

```diff
@@ -0,0 +1,48 @@
+.header {
+  position: fixed;
+  width: 100vw;
+  background-color: #e9e9e9;
+  top: 0;
+  height: 60px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: flex-end;
+
+  .logo {
+    position: absolute;
+    left: 10px;
+    top: 10px;
+    width: 48px;
+    height: 40px;
+  }
+
+  div {
```

Comment:
Really bad practice to write tag-name selector. Please, try to replace it with .class-name selector (or maybe #id-selector)